utils/__tests__/check_quickstart_uniqueness.test.js

```diff
@@ -40,8 +40,8 @@ describe('Action: check quickstart uniqueness', () => {
   test('does not find match', () => {
```

Comment:
I guess formatter hasnt been run on our repo in a while. Sorry for the extra 'fluff'

You can assume anything that doesn't touch a `quickstart` or `data source` can be ignored